package.json

```
@@ -6,13 +6,16 @@
     "start": "run-p start:local-backend start:frontend",
     "preview": "run-p start:local-backend preview:frontend",
     "test": "cd frontend && vitest",
+    "e2e": "playwright test",
+    "e2e-ui": "playwright test --ui",
```

Maybe we want to use this alias instead to be more consistent with other script names.
```suggestion
    "e2e:ui": "playwright test --ui",
```